aws/policy/storage-services.yaml

```
@@ -57,10 +57,44 @@ Statement:
       - backup:UntagResource
       - backup:UpdateBackupPlan
       - backup-storage:MountCapsule
+      - memorydb:Describe*
+      - memorydb:DeleteCluster
+      - memorydb:DeleteParameterGroup
+      - memorydb:DeleteSubnetGroup
+      - memorydb:DeleteUser
+      - memorydb:DeleteACL
+      - memorydb:List*
+      - memorydb:Describe*
+      - memorydb:CreateParameterGroup
+      - memorydb:CreateSubnetGroup
+      - memorydb:CreateUser
+      - memorydb:CreateACL
     Resource: "*"
 
   - Sid: AllowGlobalUnrestrictedResourceActionsWhichIncurFees
     Effect: Allow
     Action:
       - s3:PutMetricsConfiguration
     Resource: "*"
+
+  - Sid: AllowRegionalRestrictedResourceActionsWhichIncurFees
+    Effect: Allow
+    Action:
+      - memorydb:CreateCluster
+      - memorydb:CreateSnapshot
+    Resource:
+      - 'arn:aws:memorydb:{{ aws_region }}:{{ aws_account_id }}:cluster/*'
+      - 'arn:aws:memorydb:{{ aws_region }}:{{ aws_account_id }}:snapshot/*'
```

I tested and it requires something like this instead:

```suggestion
    Resource: "*"
    Condition:
      StringEquals:
        aws:RequestedRegion:
          - '{{ aws_region }}'
```

---

Resources must be wildcard because it wouldn't work otherwise.

---

I made the change, but I noticed that the compute.yaml file isn't using a wildcard.